WorkloadManagementSystem/Client/DownloadInputData.py

```diff
@@ -72,8 +73,8 @@ def execute( self, dataToResolve = None ):
     # determine Disk and Tape SEs
     diskSEs = set()
     tapeSEs = set()
-    for localSE in localSEList:
-      seStatus = self.__storageElement( localSE ).getStatus()['Value']
+    for localSE in [se for se in localSEList if se]:
+      seStatus = StorageElement( localSE ).getStatus()['Value']
```

Comment:
This is a good occasion to start checking the output result status before asking for Value